src/lib.rs

```diff
@@ -154,6 +173,11 @@ impl<'a> Symbol<'a> {
         self.name
     }
 
+    /// The section index of the symbol.
+    pub fn section(&self) -> usize {
```

Comment:
While both ELF and Mach-O use 0 to mean undefined, I think it would be better to change the return type to `Option<usize>`, and return `None` if the index is 0.